knimin/lib/data_access.py

```diff
@@ -2415,6 +2405,58 @@ def set_deposited_ebi(self):
                  WHERE barcode IN %s"""
         self._con.execute(sql, [barcodes])
 
+    def _study_exists(self, study_id):
+        """Cofirms that a study ID exists
+
+        Parameters
+        ----------
+        study_id : int
+
+        Raises
+        ------
+        ValueError
+            If the study ID does not exist
+        """
+        sql = """SELECT EXISTS (SELECT 1 FROM pm.study WHERE study_id = %s)"""
```

Comment:
Agree on that statement but my question is why `SELECT EXISTS (SELECT 1 FROM pm.study WHERE study_id = %s)` vs. `SELECT EXISTS (SELECT study_id FROM pm.study WHERE study_id = %s)`?